src/Security/RememberLoginHash.php

```diff
@@ -190,14 +204,21 @@ public static function generate(Member $member)
     /**
      * Generates a new hash for this member but keeps the device ID intact
      *
+     * @deprecated 5.3.0 Token renewal will be removed in 6.0.0
      * @return RememberLoginHash
      */
     public function renew()
     {
-        $hash = $this->getNewHash($this->Member());
-        $this->Hash = $hash;
+        // Only regenerate token if configured to do so
+        $replaceToken = RememberLoginHash::config()->get('replace_token_during_session_renewal');
+        if ($replaceToken) {
+            $hash       = $this->getNewHash($this->Member());
+            $this->Hash = $hash;
```

Comment:
We don't tend to align code like that, because as soon as we update the line in the future we then have to update _both_ lines to keep them aligned, and it makes diffs messy.

I don't think this is strictly in our coding standards though so I won't block merging if this change isn't made.
```suggestion
            $hash = $this->getNewHash($this->Member());
            $this->Hash = $hash;
```

Reply:
More than happy to tweak it, this was caused by overzealous IDE behaviour 😅